app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java

```diff
@@ -28,13 +28,19 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_user_profile);
+
+        // Retrieve the user this intent was started with.
+        Bundle bundle = getIntent().getExtras();
+        User user = bundle.getParcelable("user");
+
```

Comment:
Make sure this doesn't break viewing a profile normally (i.e. through the MainActivity "view profile" button).